ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credential-libraries/credential-library.js

```diff
@@ -5,13 +5,11 @@
 
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
-import { action } from '@ember/object';
 
 export default class ScopesScopeCredentialStoresCredentialStoreCredentialLibrariesCredentialLibraryRoute extends Route {
   // =services
 
   @service store;
```

Comment:
None blocking comment: I know the code was there before, but in the line 23, we are declaring `async model()` but I think we do not need `async` on that method since we are not awaiting, isn't? `findRecord` returns a promise but we aren't waiting for it. Thoughts?

Reply:
i think we should leave the model as async since even though we are not awaiting we return a promise (`findRecord`) which is still a valid use case.